tests-unit/routesUser.js

```diff
@@ -42,37 +48,98 @@ describe('user', function () {
 		});
 	}
 
-	beforeEach(function (done) {
-		dao.deleteAllUsers(function (err) {
+	function configOptions(port, path, method, body, auth, version) {
```

Comment:
You're right thx for the catch up. I'll change its signature as soon as i have 5 free mins 👍